lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -142,6 +142,10 @@ html.video-fullscreen {
       &.xblock-student_view-vertical {
         max-width: $text-width-readability-max;
       }
+
+      &.xblock-anonymous_view-vertical {
```

Comment:
Instead of making duplicates, you can just do:

```
&.xblock-student_view-vertical,
&.xblock-anonymous_view-vertical {
    max-width: $text-width-readability-max;
}
```

Reply:
Leftovers from testing with different styles for anonymous blocks.